### PR TITLE
docs: angular dev v18 page issue fix

### LINKED ISSUE #58838: The menu items are invisible in Dark Mode specifically for V18 Angular dev page
### Describe the problem that you experienced

The menu items are not visible in dark mode; they only appear on mouse hover. In light mode, their visibility is also not satisfactory. This issue occurs across all browsers.

### Enter the URL of the topic with the problem

https://v18.angular.dev/

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

_No response_

### Add a screenshot if that helps illustrate the problem

Image

### If this problem caused an exception or error, please paste it here

```true

```

### If the problem is browser-specific, please specify the device, OS, browser, and version

```true

```

### Provide any additional information here in as much as detail as you can

```true

```

### PR DESCRIPTION
**Pull request by prasantht96**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.dev application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Fixes #58838 
Issue Number: #58838 

## What is the new behavior?
Changed background color for the Menu.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/app/core/layout/navigation/navigation.component.scss [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
   &.adev-nav-primary--deprecated {
     // change nav background to indicate this is an outdated version of the docs
-    background-color: color-mix(in srgb, var(--symbolic-gray), transparent 30%);
+    background-color: color-mix(in srgb, var(--page-background), transparent 30%);
```
Comment (JeanMeche) [inline adev/src/app/core/layout/navigation/navigation.component.scss:86]: This change isn't correct, we'd like to have a difference background color for deprecated doc versions.

**Review by JeanMeche — CHANGES_REQUESTED** [review]
This change isn't correct, we'd like to have a difference background color for deprecated versions of our docs.

**Comment by prasantht96** [comment]
> Deployed adev-preview for [5600cf9](https://github.com/angular/angular/commit/5600cf9df5d8c6c92cf241341f3dbe45d7309407) to: https://ng-dev-previews-fw--pr-angular-angular-58848-adev-prev-og3vdfff.web.app
> 
> _**Note**: As new commits are pushed to this pull request, this link is updated after the preview is rebuilt._

adev-nav-primary--deprecated class is missing in this preview version.

**Comment by prasantht96** [comment]
> This change isn't correct, we'd like to have a difference background color for deprecated versions of our docs.

image
image

**Review by JeanMeche — APPROVED** [review]

**Comment by JeanMeche** [comment]
Oh my bad, you're right! 
I've added the PR to the merge queue, thank you for this !

**Comment by thePunderWoman** [comment]
@prasantht96 @JeanMeche This shows that it can't merge into `18.2.x`. I think you'll need to rebase and resolve any conflicts / re-push before we can merge.

**Comment by prasantht96** [comment]
> @prasantht96 @JeanMeche This shows that it can't merge into `18.2.x`. I think you'll need to rebase and resolve any conflicts / re-push before we can merge.

The branch was already rebased on 18.2.x, and there are no conflicts. The issue preventing the merge seems to be related to the action: cleanup label, which is currently blocking the process.

I also noticed that the request for review from @bencodezen has been removed by @JeanMeche , so that should no longer be an obstacle. If the action: cleanup label can be removed, the PR should be good to merge. Please let me know if there’s anything else I need to address!

**Comment by JeanMeche** [comment]
Jessica was right, before she added the `cleanup` label there was a target issue. It should have been LTS as we're targetting the v18 branch. We should fine now.

**Comment by thePunderWoman** [comment]
@JeanMeche Ah yeah, we did release a new version last week, didn't we? LTS is correct.

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit c6cec5eefa0da7aa0f968cacddb7bf84817961b9.

The changes were merged into the following branches: 18.2.x

**Pull request closed without merging**